manifests/p/PPSSPPTeam/PPSSPP/1.15.3/PPSSPPTeam.PPSSPP.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created with Komac v1.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: PPSSPPTeam.PPSSPP
+PackageVersion: 1.15.3
+InstallerType: inno
+ReleaseDate: 2023-05-07
+Installers:
+- Architecture: x86
+  InstallerUrl: https://www.ppsspp.org/files/1_15_3/PPSSPPSetup.exe
+  InstallerSha256: 029FACC68C3B21416831FFA057C7A60AC186342C26853F117E45F5F1F47B49AA

```

```suggestion
  InstallerSha256: 029FACC68C3B21416831FFA057C7A60AC186342C26853F117E45F5F1F47B49AA
  AppsAndFeaturesEntries:
    - DisplayVersion: 1.15.3.0
       ProductCode: PPSSPP_is1
```
To match version displayed in Apps&Features